### PR TITLE
tags view and routing

### PR DESCRIPTION
**Pull request by whichking**
resolves future issue: Add TagsView and Routing

### CODE REVIEW

File: blob_client/src/containers/posts.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
+import React from 'react'
```
Comment (mplanchard) [inline blob_client/src/containers/posts.js:1]: Was this import required?

File: blob_client/src/containers/tags.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react'
```
Comment (mplanchard) [inline blob_client/src/containers/tags.js:1]: This one, too?

File: blob_client/src/containers/posts.js (hunk 1 above)
Comment (whichking) [reply blob_client/src/containers/posts.js:1]: Yes, I was getting errors without the imports.

**Review by mplanchard — APPROVED** [review]
⚓️

**Pull request merged**